xclim/indices/_threshold.py

```diff
@@ -2078,7 +2164,7 @@ def snd_season_length(
 @declare_units(snw="[mass]/[area]", thresh="[mass]/[area]")
 def snw_season_length(
     snw: xarray.DataArray,
-    thresh: Quantified = "20 kg m-2",
+    thresh: Quantified = "4 kg m-2",
```

Comment:
I wouldn't change the default value here. It'll be removed soon, and the other index will entirely replace. It shouldn't undergo breaking changes.

Reply:
Hmm, no this is another thing completely haha. Maybe better to put this in another PR? And use the UNSET trick to keep the current value?

Reply:
The UNSET trick works!